gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java

```diff
@@ -47,6 +53,9 @@ public void apply(Project rootProject) {
                             + "Gradle version in order to use the JDK setup.",
                     GradleJdksEnablement.MINIMUM_SUPPORTED_GRADLE_VERSION));
         }
+        if (areFlowActionsSupported()) {
+            rootProject.getPluginManager().apply(ToolchainFailureFlowActionsPlugin.class);
```

Comment:
the error message will only be available for repos >= 8.6

Reply:
This is fine as internally we're almost all >=8.8